### ISSUE TITLE
SCM broke on maint-1.0

### ISSUE THREAD

**Issue by bogensch** [E3SM-Project/E3SM#3020]
running the SMS_R_Ld5.T42_T42.FSCM5A97 test on maint-1.0 provides the following error:

RUNNING TESTS:
  SMS_R_Ld5.T42_T42.FSCM5A97.cori-knl_intel
Starting CREATE_NEWCASE for test SMS_R_Ld5.T42_T42.FSCM5A97.cori-knl_intel with 1 procs
Finished CREATE_NEWCASE for test SMS_R_Ld5.T42_T42.FSCM5A97.cori-knl_intel in 0.123484 seconds (FAIL). [COMPLETED 1 of 1]
    Case dir: /global/cscratch1/sd/bogensch/acme_scratch/cori-knl/SMS_R_Ld5.T42_T42.FSCM5A97.cori-knl_intel.SCM_devXX
    Errors were:
        **ERROR: Multiple matches**

I've traced it back that this error first appears with the following commit:

commit 524f049d4a64636d2bb712d39c8c3b9cf56a122c
Merge: bf8255ce3 ce8ed40a8
Author: Wuyin Lin <wlin@bnl.gov>
Date:   Wed Jun 12 14:10:19 2019 -0700

    Merge tangq/cam/F2010-HR-compset into maint-1.0 (PR #2758)
    
    This PR created the F2010 HR compset with CMIP6 forcings. The compset name is F2010C5-CMIP6-HR.
    
     - It uses daily 0.25 degree sst_seaice data.
     - atm and lnd initial conditions from an ne120 nudging simulation
     - Classical Nucleation Theory (CNT) for ice deposition growth is turned off
     - Include atm and lnd use_case files specifically for this compset

**Comment by rljacob** [comment]
The problem is the addtional compset in cam/cime_config/config_compsets.xml.  Its needs to have its one `<compset></compset>` block.  Not included in an existing one.

**Comment by rljacob** [comment]
I'll include a fix in my PR adding compy to maint-1.0

**Comment by rljacob** [comment]
Odd that this didn't break other cases on maint-1.0.  The 2 cases under test still build fine.  I think its because as soon as a match is found in config_compsets.xml it stops processing it and if your case is before the bad line, there's no error.

**Comment by wlin7** [comment]
Thanks @bogensch and @rljacob for looking into this.  Rob, can you illuminate which additional compset has the problem? It seems to me all has its own <compset></compset> block.

**Comment by wlin7** [comment]
I can see it now. The problem occur on merging (likely during resolving conflict). The PR #2758 itself has proper <compset></compset> around the compset introduced. Thanks again @rljacob for catching it.

**Comment by rljacob** [comment]
Now I get a different error when trying to run T42_T42.FSCM5A97
```
read_initial: model parameters do not match initial dataset parameters
 Model Parameters:    plon =            1  plat =            1
 Dataset Parameters:  dlon =          128  dlat =           64
 ERROR: read_initial: model parameters do not match initial dataset parameters
eters
```
The initial data file is:  atm/cam/inic/gaus/cami_0000-09-01_64x128_L72_c031210.nc

**Comment by wlin7** [comment]
I will look into that. If single_column=.true., the data dimensions are not required to be matched.

**Comment by bogensch** [comment]
@rljacob I ran T42_T42.FSCM5A97 with the most up-to-date maint-1.0 branch on Cori and my test passed.  What machine are you running on so I can try to reproduce what you are seeing?

**Comment by wlin7** [comment]
My test was also just completed on cori-knl. It passed. A separate test case with generic EUL SCM went ok as well.  @rljacob, if the run_dir still exist, can you please check in drv_in if single_column=.true. is present?

**Comment by rljacob** [comment]
I'm actually using create_newcase and I'm on compy.  Are you guys using a script or the actual test line with _R_?
./create_newcase --case scam --compset FSCM5A97 --res T42_T42

single_column is "false" for me doing the above.

**Comment by bogensch** [comment]
ah, I see.  I was running e3sm_developer tests with _R_ and it sounds like Wuyin was running with scripts (both of which set PTS_MODE to true).  Looks like we need to explicitly declare PTS_MODE to be true in config_compsets.xml if the FSCM5A97 compset is used.  I can take care of this.

**Comment by rljacob** [comment]
Ok.  I'm going to close this because the initial problem was fixed in PR #3023
